itou/users/tests/test_models.py

```diff
@@ -451,17 +451,12 @@ def test_siae_admin_without_convention_cannot_create_siae_antenna(self):
         user = siae.members.get()
         self.assertFalse(user.can_create_siae_antenna(siae))
 
-    def test_geiq_admin_can_create_siae_antenna(self):
-        # A GEIQ never has a convention.
-        siae = SiaeWithMembershipFactory(kind=SiaeKind.GEIQ, convention=None)
-        user = siae.members.get()
-        self.assertTrue(user.can_create_siae_antenna(siae))
-
-    def test_ea_admin_can_create_siae_antenna(self):
-        # An EA never has a convention.
-        siae = SiaeWithMembershipFactory(kind=SiaeKind.EA, convention=None)
-        user = siae.members.get()
-        self.assertTrue(user.can_create_siae_antenna(siae))
+    def test_admin_ability_to_create_siae_antenna(self):
+        for kind in SiaeKind:
+            with self.subTest(kind=kind):
+                siae = SiaeWithMembershipFactory(kind=kind, membership__is_admin=True)
```

Comment:
`SiaeWithMembershipFactory()` créant tout le temps une convention, si quelqu'un venait à supprimer la condition `and parent_siae.convention is not None` je crois que les tests continuerais à passer ;).

Reply:
En plus il est juste au dessus :facepalm:, je sais pas comment j'ai fait pour le manquer...